tools/sklearn/utils.py

```diff
@@ -60,17 +59,16 @@ def find_class(self, module, name):
 
         raise pickle.UnpicklingError("global '%s' is forbidden" % fullname)
 
-    @classmethod
-    def load(self, file):
-        obj = pickle.Unpickler(file)
-        obj.find_global = self.find_class
-        return obj.load()
+
+def load_model(file):
+    return SafePickler(file).load()
+
 
```

Comment:
The old `SafePickler.load` was removed. The new `load_model` seems to be the same as `pickle.load`. The  `SafePickler.find_class` which contains the whitelist checker is not applied at all. Please double check this.